### PR TITLE
Add responsive pre-scales of award art

### PR DESCRIPTION
**Pull request by TiKevin83**
better compromise between bilinear and pixelated scaling

### CODE REVIEW

File: TASVideos/Pages/Shared/_MovieModule.cshtml [hunk 1]
```diff
@@ -38,149 +47,152 @@
...
+				@foreach (var flag in Model.Flags.Where(f => !string.IsNullOrWhiteSpace(f.IconPath)))
+				{
+					<a href="@flag.LinkPath">
+						<img class="pixelart-image" title="@flag.Name" alt="[@flag.Name]" src="/@flag.IconPath" />
+					</a>
+				}
+			</div>
+		</row>
 	</cardheader>
-    <cardbody class="px-2 py-0">
```
Comment (adelikat) [inline TASVideos/Pages/Shared/_MovieModule.cshtml:56]: What happened here? whitespace?

Comment (TiKevin83) [reply TASVideos/Pages/Shared/_MovieModule.cshtml:56]: yeah you want everything standardized on tab delimited right?

**Comment by TiKevin83** [comment]
I need to create a module/taghelper for the TierIcon side before merging this, it duplicated a lot of code.

File: TASVideos/Pages/Shared/_MovieModule.cshtml (hunk 1 above)
Comment (adelikat) [reply TASVideos/Pages/Shared/_MovieModule.cshtml]: Yes please.  If there is a way to specify tabs in .cshtml files in .editorconfig, that would be ideal, so there is no need to nag

**Review by adelikat — APPROVED** [review]

**Pull request merged**